app/assets/stylesheets/components/firm_list/_firm_list.scss

```diff
@@ -23,3 +23,19 @@
 .firm-list__item__name {
   font-weight: bold;
 }
+
+.firm-list__filter {
+  font-size: $baseline-unit*2;
+  color: $color-firm-list-border-grey;
+  margin-top: 44px;
+  margin-left: 0.5em;
+  padding: 0.5em;
+  -webkit-border-radius: 15px;
```

Comment:
We use [autoprefixer](https://github.com/moneyadviceservice/frontend/blob/master/Gemfile#L65) on the frontend so we don't need to specify these. Probably worth putting on here too.